resources/web/wwi/WebotsStreaming.js

```diff
@@ -38,6 +38,8 @@ export default class WebotsStreaming extends HTMLElement {
    * mode : x3d or mjpeg
    * broadcast: boolean
    * mobileDevice: boolean
+   * callback: function
+   * disconnectCallback: function. It need to be passed there and not in disconnect because disconnect can be called from inside the web-component
```

Comment:
Typo
```suggestion
   * disconnectCallback: function. It needs to be passed there and not in disconnect because disconnect can be called from inside the web-component
```